tools/nl/embeddings/input/base/sheets_svs.csv

```diff
@@ -3534,28 +3535,28 @@ WHO/fullv_Male,Percentage of Full Immunization Coverage Among 1 Year Old male
 WHO/fullv_Rural,Percentage of Full Immunization Coverage Among 1 Year Old in Rural Areas
 WHO/fullv_Urban,Percentage of Full Immunization Coverage Among 1 Year Old in Urban areas
 WHO/GDO_q35,Estimated Population Based Prevalence Of Depression
-WHO/LBW_PREVALENCE,Low Birth Weight Prevalence
-WHO/M_Est_smk_daily,Daily Tobacco Smoking Prevalence
+WHO/LBW_PREVALENCE,incidence rate of low birth weight
+WHO/M_Est_smk_daily,incidence rate of Daily Tobacco Smoking
 WHO/mslv_Female,Measles Immunization Coverage Among 1 year old female
 WHO/mslv_Male,Measles Immunization Coverage Among 1 year old male
 WHO/mslv_Rural,Measles Immunization Coverage Among 1 year old in Rural Areas
 WHO/mslv_Urban,Measles Immunization Coverage Among 1 year old in Urban areas
-WHO/NCD_BMI_18A,Prevalence Of Underweight Among Adults
-WHO/NCD_BMI_30A,Prevalence Of Obesity Among Adults
-WHO/NCD_PAC,Prevalence Of Insufficient Physical Activity Among Adults
+WHO/NCD_BMI_18A,incidence rate Of Underweight Among Adults
+WHO/NCD_BMI_30A,incidence rate Of Obesity Among Adults
+WHO/NCD_PAC,incidence rate Of Insufficient Physical Activity Among Adults

```

Comment:
Couple of thoughts on "incidence rates":

1.  It feels odd to use it in the context of tobacco smoking or underweight.  It seem to make more sense to use in the context of a "disease".  In these contexts, percentage or prevalence feels more natural.

2.  Also, looking at the technical meaning of the words - https://www.nimh.nih.gov/health/statistics/what-is-prevalence#part_2624 - it seems incidence rate could imply "new cases" (probability of being diagnosed with disease) vs. prevalence which implies "existing cases in population".

Reply:
updated to use "proportion of" which is generally suitable for different scenarios.